CHANGELOG.md

```diff
@@ -3,28 +3,51 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [10.0.0] - 2023-06-06
+
+### Changed
+
+- `getLocation` return type to `Result<Location, LocationError>`
+
+### Fixed
+
+- `addGeotag` and `getLocation` wrong return type that is different from the one in the API docs
+
 ## [9.2.2] - 2023-06-01
 
+### Issues
```

Comment:
Issues is not part of "[keep a changelog](https://keepachangelog.com/en/1.1.0/)"

Reply:
What is the right way to notify about the known issues?

Reply:
Fixed